internal/cli/status.go

```diff
@@ -199,3 +206,50 @@ func CheckForError(err error) Result {
 
 	return Failure
 }
+
+// Failure queues up a message, which will be displayed once
+// the status ends (using the failure format).
+func (s *Status) Failure(message string, a ...interface{}) {
+	if message != "" {
+		s.failureQueue = append(s.failureQueue, fmt.Sprintf(message, a...))
+	}
+}
+
+// Success queues up a message, which will be displayed once
+// the status ends (using the warning format).
+func (s *Status) Success(message string, a ...interface{}) {
+	if message != "" {
+		s.successQueue = append(s.successQueue, fmt.Sprintf(message, a...))
+	}
+}
+
+// Warning queues up a message, which will be displayed once
+// the status ends (using the warning format).
+func (s *Status) Warning(message string, a ...interface{}) {
+	if message != "" {
+		s.warningQueue = append(s.warningQueue, fmt.Sprintf(message, a...))
+	}
+}
+
+func (s *Status) Error(err error, message string, args ...interface{}) error {
+	err = errors.Wrapf(err, message, args...)
+
```

Comment:
Removed `err != nil` check here. Error() is called only if there is `err` so I think that check is redundant.

Reply:
`errors.Wrapf` et al handle nil error as a convenience so we could as well.